doc/rex-spec-v1.md

```diff
@@ -313,8 +313,9 @@ The UnityPackage data block contains an arbitrary pre-prepared Unity package. An
 which can then directly be used by the Unity app to be included. The data block size in the header refers to the total size of
 this block plus the string size and string length. E.g.
 
-| **size [bytes]** | **name**  | **type** | **description**                                              |
-|------------------|-----------|----------|--------------------------------------------------------------|
-| 2+sz             | assetname | string   | name of the asset stored in the blob                         |
-|                  | data      | bytes    | data of the unity asset content                              |
+| **size [bytes]** | **name**       | **type**  | **description**                                              |
+|------------------|----------------|-----------|--------------------------------------------------------------|
+| 2+sz             | assetname      | string    | name of the asset stored in the blob                         |
+| 2                | targetplatform | uint16_t  | target platform for the asset package                        |
```

Comment:
Please also add a table which outlines the values for the different platforms.

Should we remove the assetname? Is this still required by REX mobile?

Reply:
it seems that this can now be removed. Most important is that your mobile reader is able to cope with this change. @re-cheid we should actually increase the block version to 2, right?